corehq/motech/forms.py

```diff
@@ -154,6 +160,13 @@ def clean_url(self):
         url = self.cleaned_data['url']
         try:
             sanitize_user_input_url_for_repeaters(url, domain=self.domain, src='save_config')
+        except CannotResolveHost:
+            # Catching and wrapping this error means that unreachable hosts do not cause the form to be invalid.
+            # The reason this is important is because we want to accept configurations where the host has not
+            # been set up yet. Wrapping this value lets consumers check whether or not the host is recognized
```

Comment:
I don't think I'm following the reasoning here. Are you saying that we are allowing the URL to be saved? ... But we never send to it?

Reply:
We do send it, but only when an unresolvable host becomes resolvable. I was told we want to support configuring repeaters before the host is fully set up, so an unresolved host will be saved, but never sent to until it resolves properly. Can I clean this comment up to clarify that?  Also, typing this out, I'm curious if an unresolvable host is clearly conveyed to the user -- it's possible they put in a typo, and they should be aware of that.

Reply:
Circling back, the user will see `Failed! Response is: Unknown URL` if a typo was made prior to testing the connection.